src/Elmish.WPF/Elmish.WPF.fsproj

```diff
@@ -52,11 +52,11 @@
 
   <ItemGroup>
     <PackageReference Include="Elmish" Version="[3.1.0, 3.99]" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[3.1.6, 99]" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[6.0.1, 99]" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Update="FSharp.Core" Version="[4.6.2, 99]" />
+    <PackageReference Update="FSharp.Core" Version="[6.0.5, 99]" />
```

Comment:
What about using a lower version?  Another idea: What about having the version depend on the target framework (so it can be even lower than for some frameworks than for others)?

Maybe this isn't worth it.  We could probably wait until someone requests compatibility with a particular version.

Reply:
It needs to be above 6 here because I start using features that were introduced in 6 for certain type resolution, unfortunately (see the build error in #496 )

Reply:
Actually, this is probably fine.  Users probably want to use the latest version of this, and this constraint allows for that.